lib/iris/cube.py

```diff
@@ -505,6 +510,16 @@ def concatenate_cube(self, check_aux_coords=True):
             is not applied to auxilliary coordinates that span the dimension
             the concatenation is occuring along. Defaults to True.
 
+        * check_cell_measures
```

Comment:
Tiny point, but this has copied a mis-spelling : 2 'r's in 'occurring'
Also above : only 1 'l' in in 'auxiliary'

However, ancillary seems to right everywhere !
Now I just need to stop saying cell methods when I mean cell measures...